### ISSUE TITLE
Vehicle stops instead of continuing into refill route

### ISSUE THREAD

**Issue by sootysax** [Courseplay/courseplay#3064]
![fsscreen_2019_01_31_15_09_37](https://user-images.githubusercontent.com/6591048/52082351-8df64980-256a-11e9-86ad-b6d5f47ea4ee.png)

**Comment by sootysax** [comment]
https://gist.github.com/sootysax/87e24a9f4a351b20cba5a314a60be94e

**Comment by pvajko** [comment]
Why is this 4 combined courses? And where in the log should I look, there's too much going on in there.

**Comment by sootysax** [comment]
i use combined courses for refill routes..  there were two courses being run (#3053 as well) - and honestly, i feel that's why it froze up.  I seem to be getting both of these issues when other courses are running, and havent been able to reproduce them when running alone.

**Comment by pvajko** [comment]
The lowest fill level I see in that log for the Fastrac is:
2019-01-31 14:59 :30 [dbg14 lp110900] Fastrac 4220 (#1): Navigator 6000: Fill levels: HERBICIDE: 1298.9/6000.0
so it looks like the time when your problem happened is not in this log.

**Comment by sootysax** [comment]
things definately get crazy when running more than one course at a time..   it seems like every single one of my issues gets worse when 3 or 4 vehicles are running at once

**Comment by pvajko** [comment]
@sootysax don't really understand, the log you posted obviously does not contain the time period when the problem happened. What can you do is mark the time (real clock time) when the problem is happening and then before you post the log here check the timestamps in the logfile and see if the time you marked is in the log.

**Comment by pvajko** [comment]
I also had at some point a Ctrl-D shortcut to put a marker message in the log, not sure if it sill works.

**Comment by sootysax** [comment]
ok, i'm just saying, moving forward, these issues are happening when multiple courses are running..  I cannot not seem to reproduce them when one vehicle is running.   I'm hoping to get clean evidence of it in a future log.

**Comment by Satissis** [comment]
Ctrl-D don't work I believe since the current setup cant handle multi keys presses in the key binding setup.
Just an info ;)

**Comment by sootysax** [comment]
Should I copy the log while the course is running?   should i stop the course?  all courses?  same game and quit?

whats the best for you to diagnose a log?

**Comment by sootysax** [comment]
here is my latest log for this issue:  It is the Fasttrac 4220 pulling the Bredal K105.  It ran out of lime and just sits there.

**Comment by pvajko** [comment]
As long as the log actually contains the issue and you can help me to approximately locate it (which vehicle, what time) I don't care how many courses are running in it.

**Comment by sootysax** [comment]
https://gist.github.com/sootysax/bf930a4078f415ce5d614aae5fe07511

**Comment by pvajko** [comment]
Yeah, I can see it there now. Is the motor running? Does anything change when you tab into it/out of it?

**Comment by sootysax** [comment]
No, the motor has shut down, nothing happens when I tab

**Comment by sootysax** [comment]
![fsscreen_2019_02_01_15_22_35](https://user-images.githubusercontent.com/6591048/52147597-3b359400-2635-11e9-86fd-a7be53d136b7.png)

**Comment by pvajko** [comment]
Could you do me a favor and try it with this turn.lua? https://gist.github.com/pvajko/f4a1e9e62e4dcfbf795bfe422862c289
I can't test where I'm now.

**Comment by sootysax** [comment]
no change in behavior..

**Comment by pvajko** [comment]
But the tractor beam starts further away from the vehicle, right?

**Comment by sootysax** [comment]
alright, so i just ran the scenario 6 times..   it is ONLY happening when you are not in the vehicle at the time..   regardless of other courses running.

**Comment by pvajko** [comment]
Cool, thanks. Do you have a savegame I can easily reproduce this?

What is your auto engine start setting? Looks like the game shuts down the engine?

**Comment by sootysax** [comment]
Auto Engine start is off..  I'm going to throw my whole FS19 folder on google drive, as I am using a mod map

**Comment by sootysax** [comment]
https://drive.google.com/file/d/1owVNt0dzDPtvdgOj2ZNB_JOLz2cS9pw4/view?usp=sharing

**Comment by pvajko** [comment]
@sootysax thanks, that helped. I've put a band aid on it, there's a conflict between the CP and the Giants helper UI, will ask Giants next week. Check if it works with 6.1.24 or later.

**Comment by sootysax** [comment]
This seems to be working better.   The engine still shuts down, but then will restart and continue on.   Not sure if you want to close this, or further look into it so the engine shut down doesn't happen at all.
